docs/content/preview/yugabyte-cloud/cloud-connect/connect/ysql.md

```diff
@@ -1,8 +1,6 @@
-<!--
-+++
-private = true
-+++
--->
+---
```

Comment:
can you revert this change? https://deploy-preview-22280--infallible-bardeen-164bc9.netlify.app/preview/yugabyte-cloud/cloud-connect/connect-client-shell/#connect-using-a-client-shell